sycl/test-e2e/CurrentDevice/current_device.cpp

```diff
@@ -0,0 +1,74 @@
+// REQUIRES: any-device-is-cpu, any-device-is-gpu
+
+// RUN: %{build} -o %t.out
```

Comment:
I think this needs %threads_lib
```suggestion
// RUN: %{build} -o %t.out %threads_lib
```